test/test_helper.rb

```diff
@@ -89,3 +89,24 @@ def download_s3_file(bucket, key)
     ).body.read
   end
 end
+
+class ActionDispatch::IntegrationTest
+  # TODO: Add this when adding devise
+  # include Devise::Test::IntegrationHelpers
+
+  # TODO: Add this implmentation back when devise
+  # is added.
+  def sign_in!(user = nil)
+    #  @current_user = user || create(:user, :onboarded)
+    #  @current_user.confirm
+    #  sign_in @current_user
+  end
+
+  def assert_correct_page(page)
```

Comment:
We don't seem to be using this.

Reply:
No, it's copied from `website`. I'm slowly moving helper methods etc over as I touch each file.